### PR TITLE
Accept both Unix time and ISO-8601

### PR DESCRIPTION
**Pull request by Roboe**
As discussed on #1: https://github.com/WeAreFairphone/fprsdb/pull/1#pullrequestreview-548601

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ JSON properties are written in the following format: `"key" [quantifier]: valueT
       "eventName": String,
       "startDate": unix_time,
```
Comment (StefanBrand) [inline README.md]: Did you mean to keep both appearances of `startDate` and `endDate` in lines 55/57 and 56/58, respectively?

Comment (Roboe) [reply README.md]: Auch, I duplicated the keys! Sorry, that was a mistake. Gonna fix that later at home. Thanks for the heads up!

Comment (Roboe) [reply README.md]: Fixed, sorry for the delay

Comment (StefanBrand) [reply README.md]: thank you, no problem!

**Pull request merged**